library/src/main/java/com/yogeshpaliyal/comrade/library/ComradeCrashHandler.kt

```
@@ -0,0 +1,99 @@
+            recordError(exception)
+            prev?.uncaughtException(thread, exception)
+        }
+        Thread.setDefaultUncaughtExceptionHandler(newExceptionHandler)
+    }
+
+    private val myBroadcastReceiver: BroadcastReceiver = object : BroadcastReceiver() {
+        override fun onReceive(context: Context?, intent: Intent?) {
+            when (intent?.action) {
+//                IA_BACKUP_ADDED_TO_QUEUE -> mListener.backupCompleted()
+//                IA_COMPANION_SETUP_COMPLETED -> mListener.comradeIsNotConfigured()
+            }
```

detekt (1.23.8)

[warning] This when block is empty.

(detekt.empty-blocks.EmptyWhenBlock)